Automattic/newspack-blocks
# chore: remove Grunt and related files/references

### All Submissions:






### Changes proposed in this Pull Request:



Removes Grunt as a dependency and all related files and references from the repo, since we use webpack to handle all asset bundling.

Closes #219.

### How to test the changes in this Pull Request:

No functional changes, just some file cleanup. Maybe confirm that the codebase installs and builds properly with `npm ci && npm run build`.

### Other information:
